ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -425,7 +427,7 @@ export default defineComponent({
           delete localState.group.interestTotal
           delete localState.group.tenancySpecified
         }
-
+        if (oldOwner !== localState.owner) setUnsavedChanges(true)
```

Comment:
What about just setting this in the mutation?

Reply:
Just concerned this might apply to registrations as well. Also at this point, the user hasn't submit any change right?

Reply:
I have added a  check to ensure that check is only on Mhr transfer,

Reply:
Seems a little inconsistent with the other approaches, where it is just set in the mutation when a change is made. 
Is that not possible here?

Reply:
Also, there is most likely a root emit required to catch the users who use the browser back button vs just hitting cancel.